### PR TITLE
feat(common): internationalization-optimization

### PR DESCRIPTION
**Pull request by litikky**
## What this PR does / why we need it:
*feat(msp): side menu & tab
*feat(cmp): side menu & tab
*feat(admin): side menu & tab
*fix: apply allWordsFirstLetterUpper in header & button & menu & breadcrumb

## Which issue(s) this PR fixes:
Fixes #

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=304050&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwic3RhdGVzIjpbNDQwMiw3MTA0LDcxMDUsNDQwMyw0NDA0LDcxMDYsNDQwNiw0NDA3LDQ0MTIsNDUzOCw0NDEzLDQ0MTQsNDQxNSw0NDE2XSwiYXNzaWduZWVJRHMiOlsiMTAwMzE0NiJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1174&type=TASK)

## Does this PR introduce a user interface change?

✅ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       internationalization-optimization in  side menu & tab and apply allWordsFirstLetterUpper in header & button & menu & breadcrumb |
| 🇨🇳 中文    |     侧边栏及tab的国际化修复；在header、button、breadcrum应用全部单词首字母大写       |

## Need cherry-pick to release versions?
❎ No

### CODE REVIEW

File: shell/app/antd-overwrite/button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import AntdButton from 'antd/lib/button';
+import { allWordsFirstLetterUpper } from 'common/utils';
+
+const Button = React.forwardRef(({ children, ...props }: any, ref) => {
+  if (typeof children === 'string') {
+    return <AntdButton {...props}>{allWordsFirstLetterUpper(children)}</AntdButton>;
+  }
+  return <AntdButton {...props}>{children}</AntdButton>;
+}) as unknown as typeof AntdButton;
+
```
Comment (Zero-Rock) [inline shell/app/antd-overwrite/button/index.tsx:24]: Button.Group is undefined

File: shell/app/antd-overwrite/button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import React from 'react';
+import AntdButton from 'antd/lib/button';
+import { allWordsFirstLetterUpper } from 'common/utils';
+
+const Button = React.forwardRef(({ children, ...props }: any, ref) => {
```
Comment (Zero-Rock) [inline shell/app/antd-overwrite/button/index.tsx:18]: useless ref

File: shell/babel.config.js [hunk 3]
```diff
@@ -112,6 +114,7 @@ module.exports = (api) => {
             libraryName: 'antd',
             customName(name, file) {
               if (overwriteMap[name]) {
+                console.log(name, `app/antd-overwrite/${name}`);
```
Comment (Zero-Rock) [inline shell/babel.config.js:117]: remove log

File: shell/app/antd-overwrite/button/index.tsx (hunk 2 above)
Comment (litikky) [reply shell/app/antd-overwrite/button/index.tsx:18]: resolved

File: shell/babel.config.js (hunk 3 above)
Comment (litikky) [reply shell/babel.config.js]: resolved

**Pull request merged**